### ISSUE TITLE
[BUG] java.lang.string cannot be cast to java.util list when parsing Schemas

### ISSUE THREAD

**Issue by mrk-han** [willowtreeapps/conductor-mobile#145]
## The problem

Setting Schema in Config

## Environment

* Conductor-Mobile Version: 0.19.1
* Last Conductor-Mobile version that did not exhibit the issue (if applicable):n/a
* Desktop OS/version used to run Conductor-Mobile: n/a
* Mobile platform/version under test: n/a
* Real device or emulator/simulator: n/a
* `appium-doctor` results are passing [X]:

## Details

```
java.lang.ClassCastException: java.lang.String cannot be cast to java.util.List
	at com.joss.conductor.mobile.ConductorConfig.readConfig(ConductorConfig.java:148)
	at com.joss.conductor.mobile.ConductorConfig.<init>(ConductorConfig.java:101)
	at com.joss.conductor.mobile.ConductorConfig.<init>(ConductorConfig.java:78)
	at com.joss.conductor.mobile.SauceLabsListener.<init>(SauceLabsListener.java:21)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at org.testng.internal.ClassHelper.newInstance(ClassHelper.java:69)
	at org.testng.internal.DefaultListenerFactory.createListener(DefaultListenerFactory.java:13)
	at org.testng.TestRunner.initListeners(TestRunner.java:315)
	at org.testng.TestRunner.init(TestRunner.java:274)
	at org.testng.TestRunner.init(TestRunner.java:241)
	at org.testng.TestRunner.<init>(TestRunner.java:167)
	at org.testng.SuiteRunner$DefaultTestRunnerFactory.newTestRunner(SuiteRunner.java:663)
	at org.testng.SuiteRunner.init(SuiteRunner.java:260)
	at org.testng.SuiteRunner.<init>(SuiteRunner.java:198)
	at org.testng.TestNG.createSuiteRunner(TestNG.java:1295)
	at org.testng.TestNG.createSuiteRunners(TestNG.java:1273)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
```

`currentSchemes: ${CONDUCTORCURRENTSCHEMES}`

```
<build>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <version>2.20.1</version>
                <configuration>
                    <suiteXmlFiles>
                        <suiteXmlFile>${suiteXmlFile}</suiteXmlFile>
                    </suiteXmlFiles>
                    <parallel>methods</parallel>
                    <threadCount>10</threadCount>
                    <properties>
                        <property>
                            <name>surefire.testng.verbose</name>
                            <value>10</value>
                        </property>
                    </properties>
                    <reportsDirectory>${project.build.directory}/reports/${maven.build.timestamp}</reportsDirectory>
                    <systemPropertyVariables>
                        <conductorCurrentSchemes>sauce-ios-mock</conductorCurrentSchemes>
                    </systemPropertyVariables>
                </configuration>
            </plugin>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-enforcer-plugin</artifactId>
                <version>1.4.1</version>
                <configuration>
                    <rules>
                        <dependencyConvergence />
                    </rules>
                </configuration>
            </plugin>
        </plugins>
    </build>
```

![image](https://user-images.githubusercontent.com/22263679/67354511-4c0b9a00-f523-11e9-8011-ba337dc9b361.png)

**Comment by mrk-han** [comment]
Pretty sure this is the same thing @bglazierjr saw almost exactly a year ago.

**Comment by mrk-han** [comment]
![image](https://user-images.githubusercontent.com/22263679/67354883-be7c7a00-f523-11e9-8b93-7804320dee0d.png)
This will run tests but not sure it is actually parsing the schema

**Comment by mrk-han** [comment]
TL;DR is that we need to make sure we can pass in 1 or multiple schemes here, with or without env variables
